cmd/alert.go

```diff
@@ -164,11 +199,47 @@ inactive = 0`,
 
 func init() {
 	rootCmd.AddCommand(alertCmd)
+
 	fs := alertCmd.Flags()
+
+	fs.StringVarP(&cliAlertConfig.NoAlertsState, "no-alerts-state", "T", "OK", "State to assign when no alerts are found (0, 1, 2, 3, OK, WARNING, CRITICAL, UNKNOWN). If not set this defaults to OK")
+
+	fs.StringArrayVar(&cliAlertConfig.ExcludeAlerts, "exclude-alert", []string{}, "Alerts to ignore. Can be used multiple times and supports regex.")
+
 	fs.StringSliceVarP(&cliAlertConfig.AlertName, "name", "n", nil,
 		"The name of one or more specific alerts to check."+
 			"\nThis parameter can be repeated e.G.: '--name alert1 --name alert2'"+
 			"\nIf no name is given, all alerts will be evaluated")
+
 	fs.BoolVarP(&cliAlertConfig.ProblemsOnly, "problems", "P", false,
 		"Display only alerts which status is not inactive/OK. Note that in combination with the --name flag this might result in no alerts being displayed")
 }
+
+// Function to convert state to integer.
+func convertStateToInt(state string) (int, error) {
+	state = strings.ToUpper(state)
+	switch state {
+	case "OK", "0":
+		return check.OK, nil
+	case "WARNING", "1":
+		return check.Warning, nil
+	case "CRITICAL", "2":
+		return check.Critical, nil
+	case "UNKNOWN", "3":
+		return check.Unknown, nil
+	default:
+		return check.Unknown, errors.New("invalid state")
+	}
+}
+
+// Matches a list of regular expressions against a string.
+func matches(input string, regexToExclude []string) bool {
+	for _, regex := range regexToExclude {
+		re := regexp.MustCompile(regex)
```

Comment:
This is not ideal. If the regex is invalid (as hard as it might be to achieve that), this line panics.
I think I would prefere a controled exit with a helpful message here.
the `..MustCompile` functions are preferable for static expressions.